### ISSUE TITLE
SelectFields for MorphTo relation

### ISSUE THREAD

**Issue by EdwinDayot** [rebing/graphql-laravel#275]
Hi,

I'm trying to get a relation through a MorphTo relation. So for a code like following:

```php
MyModel::with('anotherModel')->get();
```

There should be an SQL request to fetch the "MyModel", and X other SQL requests for each morph types.

Of course, the `Type` itself should implement the relation, but as the `Type` could be anything, it has to be an `Interface`. Problem is that the `Interface` is not linked to any `Model`, so when the following code is executed:

```php
// SelectFields.php -> getSelectableFieldsAndRelations
$primaryKey = self::getPrimaryKeyFromParentType($parentType);
```

Referencing the following code:

```php
// SelectFields.php
private static function getPrimaryKeyFromParentType($parentType)
{
    return isset($parentType->config['model']) ? app($parentType->config['model'])->getKeyName() : null;
}
```

The return value is `null` (because of the absence of the `model` property in the config).
Since the primary key is null, the queries looks like:

```sql
select from my_morph_table;
```

Is it a bug, or am I not understanding how it should work ?

Thanks

**Comment by EdwinDayot** [comment]
After more investigation, it appears that this bug is related to #290

**Comment by mfn** [comment]
https://github.com/rebing/graphql-laravel/issues/290 has been fixed and I've just [released v1.23.0](https://github.com/rebing/graphql-laravel/releases/tag/v1.23.0) with it
